civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1102,20 +1102,20 @@ function civihr_employee_portal_views_date_fields_to_timestamp(&$view, &$query,
       // iterating single conditon structure
       foreach ($conditions as &$condition) {
 
-        $operatorEqualsFormula = isset($condition['operator']) && $condition['operator'] =='formula';
+        $operatorEqualsFormula = isset($condition['operator']) && $condition['operator'] == 'formula';
         if ( !$operatorEqualsFormula ) continue;
```

Comment:
Please change all places where you have a statement on the same time to use braces, e.g.

`if ( !$operatorEqualsFormula ) continue;`

should be 

```
if ( !$operatorEqualsFormula ) {
  continue;
}
```